grid/social-media/followers/profile.supr

```diff
@@ -1,33 +1,38 @@
+"""
+Social Media Followers
+List folowers of a social media profile (e.g. Twitter followers).
+"""
+
 name = "social-media/followers"
-version = "1.1.0"
+version = "1.1.1"
 
 """
 Get Followers
-Retrieve paginated list of social-media profile followers.
+Retrieve paginated list of followers for social media profile.
```

Comment:
missing indefinite article ...followers for **a** social media...

Reply:
```suggestion
Retrieve a paginated list of followers for a social media profile.
```